graphix/graphsim/rxgraphstate.py

```diff
@@ -81,7 +81,7 @@ def neighbors(self, node) -> Iterator:
         See :meth:`BaseGraphState.neighbors`.
         """
         nidx = self.nodes.get_node_index(node)
-        return iter(self._graph.neighbors(nidx))
+        return iter(self.nodes.idx_to_num[idx] for idx in self._graph.neighbors(nidx))
```

Comment:
`iter` might be redundant as genexp itself is iterator.

Reply:
Fixed in 33b2941. Thanks!